src/CLI/clitree/cli-xml/nat.xml

```diff
@@ -110,14 +110,14 @@ limitations under the License.
              ptype="UINT"
              >
          </PARAM>
-         <ACTION builtin="clish_nop"></ACTION>
+         <ACTION>python $SONIC_CLI_ROOT/sonic-cli-nat.py patch_openconfig_nat_nat_instances_instance_config_tcp_timeout ${tcp-timeout-value}</ACTION>
      </COMMAND>
 
      <COMMAND
          name="no tcp-timeout"
          help="Set Default TCP NAT entry aging timeout in seconds"
          >
-         <ACTION builtin="clish_nop"></ACTION>
+         <ACTION>python $SONIC_CLI_ROOT/sonic-cli-nat.py patch_openconfig_nat_nat_instances_instance_config_tcp_timeout 86400</ACTION>
```

Comment:
Wouldnt "delete' work? I would prefer default value handling to be done by backend.